dicom_utils/container/record.py

```diff
@@ -700,6 +724,46 @@ def __post_init__(self):
         if isinstance(self.laterality, str):
             object.__setattr__(self, "laterality", Laterality.from_str(self.laterality))
 
+    def __lt__(self, other: FileRecord) -> bool:
```

Comment:
Maybe we could rename this function `is_preferred_to`, and then `__lt__` could just refer to that function.